test/performance/range/gap_decorator_helper.hpp

```diff
@@ -0,0 +1,162 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+#include <chrono>
+#include <cmath>
+#include <cstring>
+#include <random>
+#include <utility>
+
+#include <benchmark/benchmark.h>
+
+#include <seqan3/alignment/aligned_sequence/aligned_sequence_concept.hpp>
+#include <seqan3/alignment/exception.hpp>
+#include <seqan3/alphabet/all.hpp>
+#include <seqan3/range/decorator/all.hpp>
+#include <seqan3/std/ranges>
+
+using namespace seqan3;
+
+#define SEQ_LEN_LONG 1<<18
+#define SEQ_LEN_SHORT 1<<12
```

Comment:
```suggestion
#define SEQAN3_LEN_SHORT 1<<12
```

All macros should begin with "SEQAN3"